pkg/runner/runner.go

```diff
@@ -134,9 +134,6 @@ func ParseFlags(cfg *Config, args []string) (*Config, error) {
 		if flagset["migrate"] || flagset["use-schema-version-lease"] {
 			return nil, fmt.Errorf("Migration flags not supported in read-only mode")
 		}
-		if flagset["install-timescaledb"] {
```

Comment:
we should not remove this completely; there should be an error if someone sets install extension to true. the if should be `if flagset["install-extension"] && cfg.InstallExtensions { return error }